src/core/pages/__mocks__/_about.md

```
@@ -2,29 +1,74 @@
+![Kontur Atlas](about-atlas-1.png)
```

Markdownlint

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)